angular-ngrx-scss/src/app/repository/components/repo-header/repo-header.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { selectedRepository } from 'src/app/state/repository';
+
+@Component({
+  selector: 'app-repo-header',
+  templateUrl: './repo-header.component.html',
+  styleUrls: ['./repo-header.component.scss'],
+})
+export class RepositoryHeaderComponent {
+  @Input() owner = '';
+  @Input() name = '';
```

Comment:
question: Do we need the owner and name coming in as inputs here? We're getting the repo from the store, which should have this information - can we not use those values instead?

Reply:
they are coming from different places, but there is a reason why. 
We are accessing that data through the route, and passing it to the repo-header via Input(), because we already had that data available, but the repo-header info needs to be accessible via the info in the Store.

That makes sense to you ?